packages/serverless-helpers/src/helpers/mergeStageParams.ts

```diff
@@ -0,0 +1,16 @@
+export const mergeStageParams = <
```

Comment:
Why do we need this function ? Frr the type inference ?
Can't we use lodash `merge` or `assign` ?

Since the file is exporting only one function with the same name, it would be better to use a default export

Reply:
Yes type inference here will prevent variables to be forgotten in one stage